packages/fuels-test-helpers/src/script.rs

```diff
@@ -0,0 +1,30 @@
+use fuel_core::service::{Config, FuelService};
+use fuel_gql_client::client::FuelClient;
+use fuel_gql_client::fuel_tx::{Receipt, Transaction};
+use fuels_contract::script::Script;
+use std::fs::read;
+
+pub async fn script_runner(bin_path: &str) -> Vec<Receipt> {
```

Comment:
The method should be called `run_script` (action verb), `script_runner` would be more something like an object.